var/spack/repos/builtin/packages/py-pygit2/package.py

```diff
@@ -14,15 +14,27 @@ class PyPygit2(PythonPackage):
     homepage = "http://www.pygit2.org/"
     pypi = "pygit2/pygit2-0.24.1.tar.gz"
 
+    version('1.4.0', sha256='cbeb38ab1df9b5d8896548a11e63aae8a064763ab5f1eabe4475e6b8a78ee1c8')
+    version('1.3.0', sha256='0be93f6a8d7cbf0cc79ae2f0afb1993fc055fc0018c27e2bd01ba143e51d4452')
+    version('0.28.2', sha256='4d8c3fbbf2e5793a9984681a94e6ac2f1bc91a92cbac762dbdfbea296b917f86')
     version('0.24.1', sha256='4d1d0196b38d6012faf0a7c45e235c208315672b6035da504566c605ba494064')
 
     extends('python')
```

Comment:
This line isn't needed, it's already added by the base class

Reply:
Version 0.24.1 is too old. Build of libgit2 0.24.1 is unable.
Version 0.28.2 needs old Python.
So  version 1.3.0 seems better as "typical" version.

Reply:
I'm talking about the `extends('python')` line

Reply:
Remove the line. Please see c4dfecf .